downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -84,6 +84,126 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced GraphQL server-side caching support
 
+#### Introduction of the `data.jsondata` package
```

Comment:
This is no longer consistent with the rest of this section. Shall we update? I guess we'll have to remove "Introduction of the "